scripts/rollup/modules.js

```diff
@@ -16,6 +16,7 @@ const importSideEffects = Object.freeze({
   'react-dom/server': HAS_NO_SIDE_EFFECTS_ON_IMPORT,
   'react/jsx-dev-runtime': HAS_NO_SIDE_EFFECTS_ON_IMPORT,
   'react-fetch/node': HAS_NO_SIDE_EFFECTS_ON_IMPORT,
+  'react-dom': HAS_NO_SIDE_EFFECTS_ON_IMPORT,
```

Comment:
Why is this new?

Reply:
We have a block of code in ReactTestUtils that is wrapped in the feature flag. Inside this block we reference an import from `react-dom`. When we remove the block, we end up with having a dangling import to `react-dom` at the top of the module without it being used. In turn, this causes Rollup to fail the build.

This will organically go away when we remove the flags entirely from OSS.